include/AL/alext.h

```diff
@@ -626,38 +621,35 @@ AL_API void AL_APIENTRY alGetBufferPtrvSOFT(ALuint buffer, ALenum param, ALvoid
 #endif
 
 #ifndef ALC_SOFT_output_mode
-#define ALC_SOFT_output_mode
+#define ALC_SOFT_output_mode 1
 #define ALC_OUTPUT_MODE_SOFT                     0x19AC
 #define ALC_ANY_SOFT                             0x19AD
-/*#define ALC_MONO_SOFT                            0x1500*/
-/*#define ALC_STEREO_SOFT                          0x1501*/
```

Comment:
Is it possible to keep the duplicate defines commented out like these? For people that look in the header can see they're part of the macro list, but can't be defined since they're already defined earlier (and for some other extensions, to see they're not empty like `AL_SOFT_source_length`).

Reply:
Of course! Added back.